bitmap-team-deuces/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-team-deuces",
+  "version": "1.0.0",
+  "description": "bitmap transforms",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gsmatth/lab-04-bitmap.git"
+  },
+  "author": "team deuces",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/gsmatth/lab-04-bitmap/issues"
+  },
+  "homepage": "https://github.com/gsmatth/lab-04-bitmap#readme",
+  "devDependencies": {
```

Comment:
Since you've specified your "scripts" > "test" tag to be "mocha" I suggest adding it to the devDependencies as well :)